ts/features/idpay/initiative/details/screens/InitiativeDetailsScreen.tsx

```diff
@@ -162,6 +166,7 @@ export const InitiativeDetailsScreen = () => {
               )}
             </View>
           </View>
+          <VSpacer size={32} />
         </ScrollView>
         {initiativeNeedsConfiguration && (
           <FooterWithButtons
```

Comment:
There is a small white gap in the lower part of the screen caused by the parent `SafeAreView`.
I'd suggest to remove the parent `SafeAreaView` and instead wrap with it the `FooterWithButtons` component.